root/components/home/About.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * @flow
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faCheckCircle} from '@fortawesome/free-solid-svg-icons';
+
+const About = (): React.Element<'section'> => {
+  return (
+    <section className="about">
+      <div className="container">
+        <div className="row">
+          <div
+            className="col-lg-6"
+          >
+            <h3
+              className="heading"
+            >
+              {l('About')}
+              {' '}
+              <span
+                className="color-purple"
+              >
+                {l('Music')}
+              </span>
+              <span
+                className="color-orange"
+              >
+                {l('Brainz')}
+              </span>
+            </h3>
+            <p>
+              {l('MusicBrainz is an open music encyclopedia ')}
+              {l('that collects music metadata and makes ')}
+              {l('it available to the public.')}
+              <br />
+              <br />
+              {l('MusicBrainz aims to be:')}
```

Comment:
It doesn't seem quite right to have this as one <p> with two <br>, I changed it to two <p> separated with one <br> which keeps them more or less at the same distance. I'd probably even drop the <br> but that might make the video frame on the right a bit too crushed